core/tauri/Cargo.toml

```diff
@@ -86,7 +86,7 @@ tokio-test = "0.4.2"
 mockito = "0.30"
 
 [features]
-default = [ "wry" ]
+default = [ "wry", "menu" ]
```

Comment:
I suppose this is not intended?

Reply:
I forgot I set that before i set the `--all-features --all-targets` on my IDE clippy args, will be reverted